### PR TITLE
Noa

### PR DESCRIPTION
**Pull request by noamu111**

### CODE REVIEW

**Comment by nadav7679** [comment]
U should eat more.

**Comment by nadav7679** [comment]
Should replace 'ima' and aba with <ima_name>, <aba_name>.
Noa and Lev is stupid and childish

**Comment by ghost** [comment]
hello, you're the bext!

**Comment by ghost** [comment]
> 
> 
> Should replace 'ima' and aba with <ima_name>, <aba_name>.
> That is stupid.

DO A FU*KING FLIPPPP!!!!!!!!!!
FROM A ROFFFFF!!!!!!

File: yaml/noa/yaml_exercise1_noa.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        dateOfBirth: 10.6.1990
+summary: >- my name is noa,
+  i love playing the piano and doing gymnastics.
+favMovies:
+  - i dont like movies
+  - i hate movies
+  - no movie
+
+
+]
```
Comment (benedery) [inline yaml/noa/yaml_exercise1_noa.yaml]: delete this

**Review by benedery — CHANGES_REQUESTED** [review]

**Review by benedery — APPROVED** [review]

**Pull request merged**